src/main/resources/migrations/common/V2024.04.22__baseline_tables.sql

```diff
@@ -207,9 +207,10 @@ CREATE TABLE prison_appointment
     prisoner_number        varchar(7) NOT NULL,  -- NOMS number
     appointment_type       varchar(40) NOT NULL,
     comments               varchar(200),
-    prison_loc_id          bigint NOT NULL,  -- from locations API
-    start_time             timestamp NOT NULL,
-    end_time               timestamp NOT NULL,
+    prison_loc_key         varchar(160) NOT NULL,  -- from locations API
+    on_day                 date NOT NULL,
```

Comment:
Note sure on the name.  Could just say start_date?

Reply:
`appointment_date` ?

Reply:
Done